spec/uuid.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { generateUUID } from '../main/core/uuid'; // Adjust the import path as necessary
+
+describe('generateUUID', () => {
+    it('should return a string', () => {
+        const uuid = generateUUID();
+        expect(typeof uuid).toBe('string');
+    });
+
+    it('should return a valid UUID format', () => {
+        const uuid = generateUUID();
+        const uuidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/;
```

Comment:
a comment as to what the expected UUID would look like would be useful. Maybe a RegEx101 link?

Reply:
Its in a spec file and UUID is a recongnised standard format so I think it kina speaks for itself.